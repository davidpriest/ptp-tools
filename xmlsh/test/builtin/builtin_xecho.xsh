# Tests of builtin xecho 
# xecho differs from echo in that it outputs XML expressions
#
# echo a blank line
xecho 
# echo a string
xecho "hi there"
# echo an XML expression 
xecho <[<foo>bar</foo>]>
# echo an xml sequence seperated by newlines (differe then echo)
xecho <[1,"hi",<foo/>]>
# echo into a sequence
xecho <[ <foo/>,1,"bar" ]> >{_seq}
xecho <[ 3,"bletch" ]> >>{_seq}
# should be 5
echo There are <[ count($_seq) ]> elements in the sequence
echo -n No Newlines:
xecho -n <[ 'in this text' ]>
echo :Between the colons









