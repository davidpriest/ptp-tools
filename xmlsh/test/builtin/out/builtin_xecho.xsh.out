
hi there
<foo>bar</foo>
1
hi
<foo/>
There are 5 elements in the sequence
No Newlines:in this text:Between the colons
